fcos/release-checklist.md

```diff
@@ -75,6 +75,29 @@ At this point, Cincinnati will see the new release on its next refresh and creat
 - [ ] Wait for all releases that will be released simultaneously to reach this step in the process
 - [ ] Go to the [rollout workflow](https://github.com/coreos/fedora-coreos-streams/actions/workflows/rollout.yml), click "Run workflow", and fill out the form
 
+<details>
+<summary>Rollout general guidelines</summary>
+
+|Risk|Day of the week|Rollout Start Time|Time allocation|
+| -------- | ------- | ------- | ------- |
+|risky| Tuesday | 2PM UTC | 72H |
+|common| Tuesday | 2PM UTC | 48H |
+|rapid| Tuesday | 2PM UTC | 24H |
+
+When setting a rollout start time ask "when would be the best time to react to 
+any errors or regressions from updates?". Commonly we select 2PM UTC so that the 
+rollout's start at 10am EST(+ or - 1 for daylight savings), but these can be
+fluid and adjust after talking with the fedora-coreos IRC.
+
+Additionally this is impacted by the day of the week or holidays, again ask 
```

Comment:
To simplify, let's move the first half before the comma into the previous paragraph, and delete the second half as redundant.